versions/development/SPEC.md

```diff
@@ -698,6 +699,48 @@ Pair values can also be specified within the [workflow inputs JSON](https://gith
 }
 ```
 
+### Optional literals
+
+Any non-optional value can be  turned into an optional value by assigning it to an optional variable:
```

Comment:
I think the distinction that's important to make here is that the value doesn't change, but the type of the value does change.

Reply:
Added this